docs/content/preview/migrate/performance.md

```diff
@@ -24,39 +24,41 @@ There are several factors that slow down data-ingestion performance in any datab
+  If CPU use is greater than 50-60%, you should lower the number of jobs. Similarly, if CPU use is low, you can increase the number of jobs.
+
+   {{< note title="Note" >}}
+
+   If timeouts are witnessed, then import should be restarted with less parallelism.
 
-- **Specify the number of table splits**:
+   {{< /note >}}
 
-  - For larger tables and indexes that are [hash](../../architecture/docdb-sharding/sharding/#hash-sharding) sharded, specify the number of initial tablet splits as a part of the table DDL statement. This can help distribute the table data across multiple nodes right from the get go. Refer to [hash-sharded tables](../../architecture/docdb-sharding/tablet-splitting/#hash-sharded-tables) for an example of how to specify the number of tablets at table creation time.
+- **Increase batch size**. If the [--batch-size](../yb-voyager-cli/#batch-size) (default is 20000) is too small, the import will run slower because the time spent importing data may be comparable or less than the time spent on other tasks, such as bookkeeping, setting up the client connection, and so on. Increasing the batch-size to a very high value is not recommended as the whole batch is executed in one transaction.
+
+- **Add disks** to reduce disk write contention. YugabyteDB servers can be configured with one or multiple disk volumes to store tablet data. If all tablets are writing to a single disk, write contention can slow down the ingestion speed. Configuring the [YB-TServers](../../reference/configuration/yb-tserver/) with multiple disks can reduce disk write contention, thereby increasing throughput. Disks with higher IOPS and better throughput also improve write performance.
 
-  - For larger tables and indexes that are [range](../../architecture/docdb-sharding/sharding/#range-sharding) sharded, if the value ranges of the primary key columns are known ahead of time, pre-split them at the time of creation. Refer to [range-sharded tables](../../architecture/docdb-sharding/tablet-splitting/#range-sharded-tables) for an example of how to specify the split points.
+- **Enabling packed columns** helps in increasing the throughput to more than two times. Enable packed columns on the YugabyteDB cluster by setting the YB-TServer gflag `ysql_enable_packed_row` to true.
 
-- **Increase cluster size**. Write contention is reduced with larger cluster sizes.
+- **Client machine's disk** with higher IOPS and better throughput also improves the import performance as the performance of splitter which splits the large data file into smaller splits of 20000 rows depends on the client machine's disk.
```

Comment:
```suggestion
- **Configure the client [what is this, the source? target?] machine's disk** with higher IOPS and better throughput to improve the performance of the splitter, which splits the large data file into smaller splits of 20000 rows. Splitter performance depends on the client machine's disk.
```

Reply:
@priyanshi-yb - It is the target machine here ?

Reply:
No, the machine on which Voyager is running.

Reply:
In that case , I'm going to go with host machine - https://docs.yugabyte.com/preview/migrate/install-yb-voyager/#prepare-the-host